source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -36,6 +36,19 @@ content: |
              if (err) throw err;
              localMasterKey = data;
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 4
+
+           from bson import binary
+           
+           const path = "./master-key.txt";
+           local_master_key =  binary.Binary(open(path, "rb").read (96))
```

Comment:
Would it make sense to be consistent with:
-using a single space after `=`
-removing the space between functions (in this case, `read`) and the parameter?

Reply:
It would make a lot of sense (actually not even sure how this ended up with the extra space). I've altered the following:

- Used a single space after = 
- Removed the extra space between functions